cmds/cpu/cpu.go

```diff
@@ -136,7 +131,23 @@ func getPort(host, port string) string {
 }
 
 func newCPU(host string, args ...string) (retErr error) {
-	// note that 9P is enabled if namespace is not empty OR if ninep is true
+	// When running over ssh, many environment variables
```

Comment:
This code for building the port and environment seems twisty and exactly the kind of thing that would benefit from being pulled out into utility functions that have some tests to make sure that the environment is sane? Maybe just add a TODO in this PR.